src/main/java/core/basesyntax/IFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface IFigure {
```

Comment:
Interface names should not be verbs. Consider renaming the interface to something that reflects what it represents or does without using a verb, such as 'Drawable' if it's meant for objects that can be drawn.